explore-analyze/report-and-share/automating-report-generation.md

```diff
@@ -159,3 +165,65 @@ If you experience issues with the deprecated report URLs after you upgrade {{kib
+
+```{applies_to}
+stack: preview 9.1
+serverless: preview
+```
+
+Save time by setting up a recurring task that automatically generates reports and shares them on a schedule that you choose. 
+
+### Requirements [scheduled-reports-reqs]
+
+* To use the scheduled reports feature, your role needs [access to reporting](../../deploy-manage/kibana-reporting-configuration.md#grant-user-access).
+* (Optional) To view and manage other users’ reports and schedules, your role needs `All` privileges for the **Manage Scheduled Reports** feature. You can set this by configuring your role's {{kib}} privileges.
+* Sharing reports outside of {{kib}} requires a default preconfigured email connector.
+
+   * **{{ech}} or {{serverless-short}} users**: You do not need to set up a default preconfigured email connector. Kibana provides you with a built-in preconfigured email connector that uses the SMTP protocol to send emails. To view it, go to the **Connectors** page and find the Elastic-Cloud-SMTP connector.
+   * **Self-managed users**: You must set up a default preconfigured email connector to send notifications outside of {{kib}}. To do this:
+     
+     1. Open your `kibana.yml` file and add a new section for the `xpack.actions.preconfigured` setting. This setting specifies configuration details that are specific to the type of preconfigured connector you're defining. 
+     2. Under the `xpack.actions.preconfigured setting`, define the email connector. For example: 
+
+        ````
+        xpack.actions.preconfigured:
+          my-email:
+          name: preconfigured-email-connector-type
+          actionTypeId: .email
+          config:
+            service: other
+            from: testsender@test.com
+            host: validhostname
+            port: 8080
+            secure: false
+            hasAuth: true
+          secrets:
+            user: testuser
+            password: passwordkeystorevalue
+         `````
+
+         :::{tip} 
+         Refer to [Email connectors](kibana://reference/connectors-kibana/pre-configured-connectors.md#preconfigured-email-configuration) to learn about requirements for different email services and providers.
+         :::
+
+### Create a schedule [create-scheduled-report]
+
+1. Open the saved Discover session, dashboard, or visualization you want to share. 
+2. Click the **Export** icon, then **Schedule export**.
+3. Enter the requested details, and (optional) enable **Print format** to generate the report in a printer-friendly format.
+4. Set up a schedule for generating the report.
+
+    * **Date**: Choose when to start generating reports.
+    * **Timezone**: Specify a timezone for the schedule.
+    * **Repeat**: Choose how often you want to generate reports.  
+
+5. (Optional) To share generated reports outside of Kibana, enable **Send by email** and enter a list of email addresses. Recipients will receive emails with the generated reports attached and on the schedule that you specified.
```

Comment:
We should mention that users without `Manage Scheduled Reporting` feature privilege will only be able to send the report to themselves.

Reply:
Added in [c7e576f](https://github.com/elastic/docs-content/pull/2118/commits/c7e576f9c7b37c8cbe8cbfa140ca33ebed3cc42d).